### PR TITLE
Added title and admin can edit excerpts from SDG pages

### LINKED ISSUE #81: As an admin I want to be able to edit the SDG text

### LINKED ISSUE #85: As a user I want the name of my blog to be the type of activity + title
Acceptance Criteria:
GIVEN on an SDG page or all SDG page
WHEN showing a blog
THEN The title of it should be the type of activity + the title of the blog.

### PR DESCRIPTION
**Pull request by avanheeren**

### CODE REVIEW

**Review by Sanyi0001 — APPROVED** [review]
Nicely done

File: resources/views/blogs/create.blade.php [hunk 1]
```diff
@@ -151,6 +151,23 @@
...
 
+        <div class="field is-horizontal">
+            <div class="field-label is-normal">
+                <label class="label">Title</label>
+            </div>
+            <div class="field-body">
+                <div class="field">
+                    <div class="control">
+                        <input class="input @error('title') is-danger @enderror" name="title" type="text"
+                               placeholder="Title of your project" value="{{old('title')}}">
```
Comment (Koreyoshi159) [inline resources/views/blogs/create.blade.php]: Spacing

File: resources/views/blogs/edit.blade.php [hunk 2]
```diff
@@ -148,6 +148,23 @@
...
 
+        <div class="field is-horizontal">
+            <div class="field-label is-normal">
+                <label class="label">Title</label>
+            </div>
+            <div class="field-body">
+                <div class="field">
+                    <div class="control">
+                        <input class="input @error('title') is-danger @enderror" name="title" type="text"
+                               placeholder="the title of your project" value="{{$blog->title}}">
```
Comment (Koreyoshi159) [inline resources/views/blogs/edit.blade.php]: Spacing

**Review by Koreyoshi159 — APPROVED** [review]
nice work

File: resources/views/blogs/create.blade.php (hunk 1 above)
Comment (AnnevanHeeren) [reply resources/views/blogs/create.blade.php:162]: where exactly?

**Review by AlexGit123 — APPROVED** [review]
Nice

**Review by koenhamstra — APPROVED** [review]
Looks good

**Pull request merged**